libsecret/libsecret.json

```
@@ -2,6 +2,7 @@
   "name": "libsecret",
   "buildsystem": "meson",
   "config-opts": [
+    "-Dcrypto=gnutls",
```

This needs gnutls >= 3.8.2, the runtime ships 3.8.0 it silently falls back to libgcrypt.

---

https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/merge_requests/17826

---

Please propose this change in a separate MR.

---

> Please propose this change in a separate PR.

Done